Unix/nits/base/nits.h.bkp

```diff
@@ -873,23 +873,13 @@ NITS_EXPORT ptrdiff_t NITS_PRESENCE;
 
```

Comment:
Why do we need a backup copy of nits.h?  It looks like Bruce created this.  Check with him, and delete this file.

Reply:
I will talk to Bruce to know why.